### DISCUSSION TITLE
protection against accidental page closure / reloads?

### DISCUSSION THREAD

**Discussion by catharsis71** [Rudokhvist/ASF-STM#30] [General]
I just accidentally nuked a 5-hour-deep scan for the 3rd time today

could you possibly add a window.onbeforeunload (or something along those lines) to do a warning message if there's a scan running or finished?

**Comment by Rudokhvist** [comment]
That's a valid suggestion, I will add that to TODO list. But don't expect me to be fast with implementation, I don't have much time for pet projects recently.
